src/pudl_rmi/coordinate.py

```diff
@@ -206,6 +207,10 @@ def deprish_to_ferc1(
             clobber_plant_parts_eia (boolean): Generate and cache a new interim
                 output of the EIA plant part list and generate a new version of
                 the depreciaiton to FERC1 output. Default is False.
+            clobber_deprish : True if you want to regenerate the depreciation
+                data whether or not the output is already pickled. The
+                deprecaition data is an interim input to make the connection
```

Comment:
Depreciation is spelled wrong here

Reply:
ha i found two others spelled that way... ugh i can't spell.